### ISSUE TITLE
[feature] insert (pandoc) citekey   in the text

### ISSUE THREAD

**Issue by danieltomasz** [vyleung/logseq-logtero-plugin#12]
While writing academic text it would be useful to have a way to insert pandoc citation of an article into the text.

for example I would like to have the paragraph below

```  
Wrting an article usually involve citing many authors [@harvey2022] 
```
Using pandoc I  can further render this document.

**Comment by vyleung** [comment]
would this be done via a slash command? (potential workflow:`/insert pandoc citation` → search for and select item → citation gets inserted inline)

**Comment by danieltomasz** [comment]
indeed, this would be the most obvious worflow (and hotkey in addition maybe)
